components/blocks/ResendConfirmationBar/ResendConfirmationBar.vue

```diff
@@ -0,0 +1,71 @@
+<script setup lang="ts">
+import { ref, computed } from 'vue'
+import BaseButton from 'elements/buttons/BaseButton/BaseButton.vue'
+import useResendAccountConfirmation from 'composables/api/useResendAccountConfirmation'
+
+const currentUser = await useCurrentUser()
+const errorText = ref('')
+const showErrorText = ref(false)
+const { showAlert } = useModalAlert()
+const override = ref(false)
+const unconfirmed = computed(
+  () => !override.value && currentUser.data.value?.role === 'Registered',
+)
+
+async function resend() {
+  await useResendAccountConfirmation({
+    onError: () => {
+      errorText.value =
+        'Something went wrong. Reach out to support if the problem persists'
+      showErrorText.value = true
+    },
+    onOkay: () => {
+      showAlert({
+        body: 'If you have a valid account with us, you should receive an email from us soon.',
```

Comment:
The reason is that I forgot. I honestly think we shouldn't worry about localization until after MVP, but it might be a good idea to at least make entries for them, even if English is the only language we have for it so far.